doc/source/dev/contributor/continuous_integration.rst

```diff
@@ -119,3 +119,22 @@ It is not advised to use cibuildwheel to build scipy wheels on your own system
 as it will automatically install gfortran compilers and various other
 dependencies. Instead, one could use an isolated Docker container to build
 Linux wheels.
+
+Docker image builds
+===================
+
+Two Docker images are built in CI with GitHub Actions, and made available on
+Docker Hub under the ``scipy`` organisation:
+
+1. ``scipy/scipy-dev`` - contains all the dependencies needed for developing
+   SciPy. This image is only rebuilt when ``environment.yml`` in the root of
+   the repo is updated,
+2. ``scipy/scipy-gitpod`` - builds on top of ``scipy-dev`` and contains a built
+   development version of SciPy and the SciPy docs. It is the image pulled when
+   `a user opens Gitpod <quickstart_gitpod.html>`_.
+
+This diagram explains the CI setup, triggers, and flow of artifacts to Docker
+Hub and Gitpod:
+
+.. image:: ../../_static/gitpod/gitpod_ci_build_flow.png
+    :alt: Diagram of how Docker images are built for Docker Hub and Gitpod in CI
```

Comment:
```suggestion
    :alt: Diagram of how Docker images are built for Docker Hub and Gitpod in CI
    
.. warning:: These Docker images are intended for SciPy development _only_. These should not be used "as-is" for any production-level applications (in which case neither SciPy nor the maintainers can be held responsible for any misuse or configuration). 
For more details on warranty and liability, please refer to the terms of the `BSD-3 license <https://github.com/scipy/scipy/blob/main/LICENSE.txt>`_ under which SciPy is distributed.
```

Reply:
This is great thank you Tania 🙇

Reply:
I've taken over the warning, toned down a bit (explicit referencing of license or disclaiming responsibility for misuse is never a good idea - that's what the license file itself is for).